src/control/package.xml

```diff
@@ -6,14 +6,18 @@
   <description>TODO</description>
   <maintainer email="oregon.robotics.team@intel.com">Oregon Robotics Team</maintainer>
```

Comment:
Do you think having a generic e-mail account like this is worth pursuing? It could provide some stability as team members change. Eventually, if Intel no longer supported the project, it could be an account with an automatic response.

Reply:
I have mixed feelings on that. It seems like a good idea, but none of the other projects do that.